app/graphql/types/query_type.rb

```diff
@@ -111,6 +113,21 @@ def random_media(media_type:, age_ratings:)
     media
   end
 
+  field :random_user_media, Types::Interface::Media, null: true do
```

Comment:
`:random_library_media`? Thoughts?

Reply:
hmmm... I think user makes more sense??

Reply:
How about putting it under the Library type?

```graphql
query {
  currentAccount {
    profile {
      library {
        randomMedia(status: PLANNED)
      }
    }
  }
}
```

Reply:
hmm hmm, I'm not sure if I like that or not. It is in an understandable place (but might be to hidden).

Reply:
I think putting it under the library scope is a good idea.

Reply:
Also can status be an array too

Reply:
yeah, it is, nuck just made a mistake in the example.

Reply:
I was just looking at the wrong code snippet (the Media type response instead of the argument 😭)